gluten-ut/src/test/scala/org/apache/spark/sql/GlutenJoinSuite.scala

```diff
@@ -17,13 +17,29 @@
 
 package org.apache.spark.sql
 
+import org.apache.spark.sql.internal.SQLConf
+
 class GlutenJoinSuite extends JoinSuite with GlutenSQLTestsTrait {
 
   override def testNameBlackList: Seq[String] = Seq(
-    GlutenTestConstants.IGNORE_ALL,
+    // Below tests are to verify operators, just skip.
+    "join operator selection",
```

Comment:
Could you please to overwrite these UTs instead of skipping?

Reply:
Previously, have checked with Binwei, can skip for this Operator check UTs

Reply:
Curiously, why these operators should be skipped instead of fallback?

Reply:
No. Can skip the UT. Not skip the operators.  The output is match, but the operator maybe not. So Skip the operator assert UTs.